spec/dummy/config/puma.rb

```diff
@@ -1,10 +1,12 @@
+# frozen_string_literal: true
+
 # Puma can serve each request in a thread from an internal thread pool.
 # The `threads` method setting takes two numbers: a minimum and maximum.
 # Any libraries that use thread pools should be configured to match
 # the maximum value specified for Puma. Default is set to 5 threads for minimum
 # and maximum; this matches the default thread size of Active Record.
 #
-max_threads_count = ENV.fetch("RAILS_MAX_THREADS") { 5 }
+max_threads_count = ENV.fetch("RAILS_MAX_THREADS", 5)
```

Comment:
Hasn't the recommendation from rails changed from 5 to 3?

Reply:
It's irrelevant, we're never going to run a dummy app. I just let the templates do their thing.